OCPBUGS#29279: Doc improvements (#71446)
openshift/openshift-docs: modules/lvms-scaling-storage-expand-pvc.adoc

:_mod-docs-content-type: PROCEDURE
[id="lvms-scaling-expand-pvc_{context}"]
= Expanding a persistent volume claim

After scaling up the storage of a cluster, you can expand the existing persistent volume claims (PVCs). 

> To exapnd a PVC, you must update the `storage` field in the `PersistentVolumeClaim` custom resource (CR) of the PVC.

Reviewer: ```suggestion
To expand a PVC, you must update the `storage` field in the `PersistentVolumeClaim` (PVC).
```

Author: Updated as per the suggestion.

:_mod-docs-content-type: PROCEDURE
[id="lvms-scaling-expand-pvc_{context}"]
= Expanding a Persistent Volume Claim

After scaling up the storage of clusters by adding new devices, you can expand the existing Persistent Volume Claims (PVCs). 

To exapnd a PVC, you must update the `storage` field in the `PersistentVolumeClaim` custom resource (CR) of the PVC.

.Prerequisites

* Dynamic provisioning is used.
* The `StorageClass` object associated with the PVC has the `allowVolumeExpansion` field set to `true`.

.Procedure

. Log in to the OpenShift CLI (`oc`).
+
[IMPORTANT]
====
If you are using {lvms} in {rh-rhacm}, log in to the {rh-rhacm} CLI using your {product-title} credentials.
====

. Update the value of the `.spec.resources.requests.storage` field to a value that is greater than the current value by running the following command:
+
[source,terminal]
----
$ oc patch <pvc_name> -n <application_namespace> -p \ <1>

> '{ "spec": { "resources": { "requests": { "storage": "<desired_size>" }}}}' <2>

Reviewer 2: ```suggestion
'{ "spec": { "resources": { "requests": { "storage": "<desired_size>" }}}}' --type=merge
 <2>
```

Author: Updated as per the suggestion.

----
<1> Replace `<pvc_name>` with the name of the PVC that you want to expand.
<2> Replace `<desired_size>` with the new size to expand the PVC.

.Verification

. To verify that resizing is completed, run the following command:
[source, terminal]
----

> $ TBD

Reviewer 2: ```suggestion
$ oc get pvc <pvc_name> -o=jsonpath={.status.capacity.storage}
```

Author: Updated as per the suggestion.
